examples/fetch-embed-job-example/20_clean_and_embed_data.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021-2024 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import csv
+import logging
+import re
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import WordNetLemmatizer
+from sentence_transformers import SentenceTransformer
+from vdk.api.job_input import IJobInput
+
+nltk.download("stopwords")
+nltk.download("wordnet")
+nltk.download("punkt")
+
+log = logging.getLogger(__name__)
+
+
+def clean_text(text):
```

Comment:
Write a function description mainly why this is needed. 
Not so much "what" (which is clear).

Reply:
addressed